recipes/arraylake/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "arraylake" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake-{{ version }}.tar.gz
+  sha256: 663efad630aa04ebe47d5cf6903494b26e81177164e223cf51059678ff7561f3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - arraylake = arraylake.cli.main:app
+    - al = arraylake.cli.main:app
+
+requirements:
+  host:
+    - python >=3.8,<3.12
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - pip
+  run:
+    - python >=3.8,<3.12
+    - aiobotocore >=2.4,<3
+    - zarr >=2.14,<=2.16
+    - numpy >=1.20,<2
+    - donfig >=0.7,<1.0
+    - typing-extensions >=4.7.0,<5.0.0
+    - pydantic >=1.10,<2
+    - httpx >=0.23,<1.0
+    - cachetools >=5.3,<6
+    - structlog >=23.1,<24
+    - email_validator >=2,<3
+    - click >=8.1,<9
+    - rich >=12.6,<14.0
+    - typer >=0.6,<1.0
+    - pyyaml >=6,<7.0
+    - ruamel.yaml >=0.17,<1.0
+    - python-dateutil >=2,<3
+    - uvloop >=0.17,<1  # [unix]
+    - __unix  # [unix]
```

Should we have a Windows constraint to ensure the UNIX package always ships with `uvloop` (as opposed to potentially dropping it)?

```suggestion
    - uvloop >=0.17,<1  # [unix]
    - __unix  # [unix]
    - __win   # [win]
```